modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -0,0 +1,30 @@
+import Base from '../../../../../../assets/dev/js/frontend/handlers/base';
+
+export default class NestedAccordion extends Base {
+	getDefaultSettings() {
+		return {
+			selectors: {},
+			classes: {
+				active: 'e-active',
+			},
+		};
+	}
+
+	onInit( ...args ) {
+		super.onInit( ...args );
+
+		this.interlaceContainers();
+	}
+
+	interlaceContainers() {
+		if ( elementorFrontend.isEditMode() ) {
```

Comment:
```
if ( ! elementorFrontend.isEditMode() ) {
    return;
}
```
@davseve Rather use an early return here.

Reply:
Maybe we can add the function in the condition